docs/sources/installation/configuration.md

```diff
@@ -742,6 +742,65 @@ Set to true if you want to test alpha plugins that are not yet ready for general
 
 Keys of alpha features to enable, separated by space. Available alpha features are: `transformations`
 
+## [tracing.jaeger]
+
+Configure Grafana's Jaeger client for distributed tracing.
+
+You can also use the standard `JAEGER_*` environment variables to configure
+Jaeger. See the table at the end of https://www.jaegertracing.io/docs/1.16/client-features/
+for the full list. Environment variables will override any settings provided here.
+
+### address
+
+The host:port destination for reporting spans. (ex: `localhost:6381`)
+
+May be set with the environment variables `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT`.
+
+### always_included_tag
+
+Comma-separated list of tags to include in all new spans, e.g. `tag1:value1,tag2:value2`
+
+May be set with the environment variable `JAEGER_TAGS` (use `=` instead of `:` with the env var).
+
+### sampler_type
+
+Default value is `const`
+
+Specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `remote`
+
+See https://www.jaegertracing.io/docs/1.16/sampling/#client-sampling-configuration for details on the different types.
+
+May be set with the environment variable `JAEGER_SAMPLER_TYPE`.
+
+### sampler_param
+
+Default value is `1`
+
+This is the sampler configuration parameter. Depending on the value of `sampler_type`, it can be `0`, `1`, or a decimal value in between.
+
+- for `const` sampler, `0` or `1` for always `false`/`true` respectively
+- for `probabilistic` sampler, a probability between `0` and `1.0`
+- for `rateLimiting` sampler, the number of spans per second
+- for `remote` sampler, param is the same as for `probabilistic`
+  and indicates the initial sampling rate before the actual one
+  is received from the mothership
+
+May be set with the environment variable `JAEGER_SAMPLER_PARAM`.
+
+### zipkin_propagation
+
+Default value is `false`
+
+Whether or not to use Zipkin span propagation (`x-b3-` HTTP headers).
```

Comment:
This is not a complete sentence, I do not understand what you mean here. Are you saying that the value of this config entry governs whether or not to use the span propagation and that the default is not to?

Reply:
I didn't write it, it's from `defaults.ini`; however, this controls whether or not to use Zipkin's span propagation format. The default is to use Jaeger's span propagation format.

I'll clarify and update!